staker/block_validator.go

```diff
@@ -1030,6 +1033,9 @@ func (v *BlockValidator) Reorg(ctx context.Context, count arbutil.MessageIndex)
 // Initialize must be called after SetCurrentWasmModuleRoot sets the current one
 func (v *BlockValidator) Initialize(ctx context.Context) error {
 	config := v.config()
+	if config.RedisValidationClientConfig.Enabled() && v.execSpawner == nil {
+		return nil
+	}
```

Comment:
we still need to execute the next "switch" statement to set currentModuleRoot.
If execSpawner==nil and currentModuleRoot is "latest" we should return an error.

Reply:
Done.